src/main/java/brave/opentracing/BraveTracer.java

```diff
@@ -56,27 +57,28 @@
    * Returns an implementation of {@linkplain io.opentracing.Tracer} which delegates
    * the the provided Brave Tracer.
    */
-  public static BraveTracer wrap(brave.Tracer brave4) {
+  public static BraveTracer create(Tracing brave4) {
     return newBuilder(brave4).build();
   }
 
-  public static Builder newBuilder(brave.Tracer brave4) {
+  public static Builder newBuilder(Tracing brave4) {
     return new Builder(brave4);
   }
 
   public static final class Builder {
-    brave.Tracer brave4;
+    Tracing brave4;
     Map<Format<TextMap>, Propagation<String>> formatToPropagation = new LinkedHashMap<>();
 
-    Builder(brave.Tracer brave4) {
-      if (brave4 == null) throw new NullPointerException("brave tracer == null");
+    Builder(Tracing brave4) {
+      if (brave4 == null) throw new NullPointerException("brave tracing component == null");
       this.brave4 = brave4;
-      formatToPropagation.put(Format.Builtin.HTTP_HEADERS, Propagation.B3_STRING);
```

Comment:
yep Propagation.B3_STRING is still there and likely will stay there. you can't always be sure of Tracing.propagation(), as it can be changed (you can of course look at its type).